README.md

```diff
@@ -268,22 +267,23 @@ From the root project directory: `python -m unittest discover -s tests`
 * Danila Ganchar, [@d-ganchar](https://github.com/d-ganchar)
 * Mikhail, [@glader](https://github.com/glader)
 * Alexander Chashnikov, [@ne1r0n](https://github.com/ne1r0n)
+* Simone Brundu, [@saimon46](https://github.com/saimon46)
 
 
 [airflow]: https://airflow.apache.org/
 [ch-driver]: https://github.com/mymarilyn/clickhouse-driver
 [ch-driver-docs]: https://clickhouse-driver.readthedocs.io/en/latest/
 [ch-driver-execute]: https://clickhouse-driver.readthedocs.io/en/latest/quickstart.html#selecting-data
 [airflow-base-op]: https://airflow.apache.org/docs/stable/_api/airflow/models/baseoperator/index.html
-[airflow-base-hook]: https://airflow.apache.org/docs/stable/_api/airflow/hooks/base_hook/index.html
+[airflow-dbapihook]: https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/hooks/dbapi/index.html
```

Comment:
Don't forget to revert here to the base hook.

Reply:
I've updated the README